taxi_service/urls.py

```diff
@@ -18,8 +18,11 @@
 from django.conf import settings
 from django.conf.urls.static import static
 
+from debug_toolbar.toolbar import debug_toolbar_urls
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    path("accounts/", include("django.contrib.auth.urls")),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + debug_toolbar_urls()
```

Comment:
The inclusion of `debug_toolbar_urls()` in the `urlpatterns` should be conditional based on the `DEBUG` setting. This is to ensure that the debug toolbar is only available in development and not in production, as it can expose sensitive information.